src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -119,10 +152,15 @@ public class CoinstakeContext
         // Default for -blockmintxfee, which sets the minimum feerate for a transaction in blocks created by mining code
         public const int DefaultBlockMinTxFee = 1000;
 
-        // Default for -blockmaxsize, which controls the maximum size of block the mining code will create
+        // Default for -blockmaxsize, which controls the maximum number of serialized bytes in a block the mining code will create.
         public const int DefaultBlockMaxSize = 750000;
```

Comment:
when you make any such modifications, feel free to improve the comment style to be xmldoc, i.e. `/// <summary> ...`

and about the change itself - i don't really get it, the constant is called DefaultBlockMaxSize, so it seems like it is a maximum size, so is it not (then we should rename it in the first place)?